web-client/src/styles/overrides.scss

```diff
@@ -11,6 +11,12 @@ option:first-child {
   font-weight: initial;
 }
 
+#inline-select {
```

Comment:
Nit: Does this need to be in overrides? Is there a more specific stylesheet that could be used?

Reply:
Actually I can't find a better place either

Reply:
We added it to overrides because these styles technically 'override' the USWDS styling in the `select` element! Is there somewhere that would make more sense? Happy to move it!

Reply:
Oops I didnt see your second comment until now lol.. ok np! TY for reviewing!